stubs/2.7/builtins.pyi

```diff
@@ -37,7 +37,10 @@ class type:
     __module__ = ''
     __dict__ = ...  # type: Dict[unicode, Any]
 
+    @overload
     def __init__(self, o: object) -> None: ...
+    @overload
+    def __init__(self, name: str, bases: tuple, dict: Dict[str, Any]) -> None: ...
```

Comment:
@JukkaL Except that right now, you have documented in various places that `Sequence[T]` is the way to do a homogenous tuple.

What we *really* need is `Intersection[tuple, Sequence[T]]` or something.